ms/MSOper/MSMetaData.cc

```diff
@@ -2128,6 +2130,30 @@ vector<String> MSMetaData::getSpwNames() const {
     return out;
 }
 
+vector<int> MSMetaData::getSpwReceiverBands() const {
+    std::set<uInt> avgSpw, tdmSpw, fdmSpw, wvrSpw, sqldSpw;
+    const auto props = _getSpwInfo(
+        avgSpw, tdmSpw, fdmSpw, wvrSpw, sqldSpw
+    );
+    vector<int> out;
+    for (const auto spw: props) {
+        out.push_back(spw.rb);
+    }
+    return out;
+}
+
+vector<int> MSMetaData::getSpwSubwindows() const {
+    std::set<uInt> avgSpw, tdmSpw, fdmSpw, wvrSpw, sqldSpw;
```

Comment:
Please document somewhere the concept of SPW subwindows. This is not a standard MS feature.